[2022-11-22 05:04:59,630] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.load_files.extract_geography_info manual__2022-11-22T05:04:43.707828+00:00 [queued]>
[2022-11-22 05:04:59,675] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.load_files.extract_geography_info manual__2022-11-22T05:04:43.707828+00:00 [queued]>
[2022-11-22 05:04:59,676] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 05:04:59,678] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-22 05:04:59,679] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 05:04:59,748] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_files.extract_geography_info> on 2022-11-22 05:04:43.707828+00:00
[2022-11-22 05:04:59,762] {standard_task_runner.py:52} INFO - Started process 410 to run task
[2022-11-22 05:04:59,778] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'load_files.extract_geography_info', 'manual__2022-11-22T05:04:43.707828+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmprnh_ixvg', '--error-file', '/tmp/tmp6_uisgu5']
[2022-11-22 05:04:59,783] {standard_task_runner.py:80} INFO - Job 218: Subtask load_files.extract_geography_info
[2022-11-22 05:05:00,048] {task_command.py:369} INFO - Running <TaskInstance: etl_solar.load_files.extract_geography_info manual__2022-11-22T05:04:43.707828+00:00 [running]> on host 8a647070b091
[2022-11-22 05:05:00,438] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=load_files.extract_geography_info
AIRFLOW_CTX_EXECUTION_DATE=2022-11-22T05:04:43.707828+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-22T05:04:43.707828+00:00
[2022-11-22 05:05:00,489] {base.py:68} INFO - Using connection ID 'mysql_solar' for task execution.
[2022-11-22 05:05:00,649] {python.py:173} INFO - Done. Returned value was: None
[2022-11-22 05:05:00,705] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=load_files.extract_geography_info, execution_date=20221122T050443, start_date=20221122T050459, end_date=20221122T050500
[2022-11-22 05:05:00,797] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-22 05:05:00,986] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
